File: frontend/src/app/model/statusmodel.ts
export class StatusModel {
  id: string;
  status: string;

  static cloneBase(status: StatusModel): StatusModel {
    const clonedStatusModel: StatusModel = new StatusModel();
    clonedStatusModel.id = status.id;
    clonedStatusModel.status = status.status;
    return clonedStatusModel;
  }
}

File: fapi/src/main/java/com/netcracker/edu/project/fapi/model/StatusModel.java
package com.netcracker.edu.project.fapi.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class StatusModel {

    private int id;
    private String status;

    public StatusModel() {
    }

    public StatusModel(int id, String status) {
        this.id = id;
        this.status = status;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}
